### PR TITLE
Added the option to automatically launch Hearthstone when HDT is launched

### PR DESCRIPTION
**Pull request by eric-garza**
This option goes together well with "Start minimized" and "Close with Hearthstone" for a more automated experience. Uses the existing functionality in Hearthstone_Deck_Tracker.Helper.StartHearthstoneAsync(). Disabled by default.

### CODE REVIEW

**Comment by Epix37** [comment]
Could you remove the "new line at end of file" changes? Other than that this looks good.

**Comment by Epix37** [comment]
One more thing actually: `No more than one change per commit` doesn't have to be taken quite as literally :). 

Could you squash the first 3 commits into something like `added config setting for StartHearthstoneWithHDT` and reword the last to say `launch hearthstone on start` rather than `start hearthstone`.

File: Hearthstone Deck Tracker/Core.cs [hunk 1]
```diff
@@ -101,6 +101,11 @@ public static void Initialize()
 					                            "This is either your first time starting HDT or the log.config file has been updated. Please restart Hearthstone, for HDT to work properly.");
 				}
 				LogReaderManager.Start(Game);
+				
+				if(Config.Instance.StartHearthstoneWithHDT && !Game.IsRunning)
```
Comment (Epix37) [inline Hearthstone Deck Tracker/Core.cs]: `Game.IsRunning` is currently only updated in `Core.UpdateOverlayAsync` and therefore always false at this point.

**Comment by eric-garza** [comment]
I have made the requested changes:

- Commits squashed and verbiage updated.
- The `Game.IsRunning` check has been moved to after the call to `Core.UpdateOverlayAsync`.
- Added new lines at the end of the source files.

**Comment by eric-garza** [comment]
I have made the requested changes:

- Commits squashed and verbiage updated.
- The `Game.IsRunning` check has been moved to after the call to `Core.UpdateOverlayAsync`.
- Fixed "no new line" warnings at the end of the source files.

**Comment by Epix37** [comment]
Perfect! Merged.

**Pull request closed without merging**